quizinstruction.css

```diff
@@ -0,0 +1,35 @@
+.quiz-info-item{
+    display: flex;
+    justify-content: space-around;
+}
+.quiz-info{
+    margin-left: 10%;
+    margin-right: 1rem;
+}
+.quiz-info-container .image-container{
+    margin: 0rem 2rem;
+    max-height: 300px;
+    max-width: 400px;
```

Comment:
use rem here

Reply:
Actually i don't want it to change according to font size that is why i have used px